packages/cli/src/lib/bundler/packageDetection.ts

```diff
@@ -116,13 +120,16 @@ async function writeDetectedPackagesModule(
     )
     .join(',');
 
-  await fs.writeFile(
-    joinPath(
-      cliPaths.targetRoot,
-      'node_modules',
-      `${DETECTED_MODULES_MODULE_NAME}.js`,
+  writeQueue.clear();
```

Comment:
Did you check that this doesn't also immediately detach/"orphan" any ongoing promises?

Could just as well skip this line actually if it's an undefined behavior (the docs don't really say clearly), not sure it'll ever truly matter.

Reply:
Yep it's not clear so might as well not clear